src/main/ui/chat-ui.gd

```diff
@@ -0,0 +1,264 @@
+extends Control
+"""
+Displays UI elements for chatting.
+
+The chat window is decorated with objects in the background called 'accents'. These accents can be injected with the
+set_accent_def function to configure the chat window's appearance.
+"""
```

Comment:
This class as a whole is getting pretty big; I've left some _(refactor)_ comments. You might take an hour to see if you can push some of this code into child objects such as ChatRect and ChatLabel. If not, I've put a refactoring task on the wiki. (Delete that refactoring task if you end up refactoring.)

Reply:
Pushed code into child objects.